prettier.config.js

```diff
@@ -10,5 +10,18 @@ module.exports = {
   trailingComma: 'all',
   bracketSpacing: false,
   jsxBracketSameLine: true,
-  parser: 'flow',
+  overrides: [
+    {
+      files: '*.js',
+      options: {
+        parser: 'flow',
```

Comment:
Use Flow only for JS so that other filetypes can be formatted by Prettier using their respective parsers.

Reply:
Are there other files that should be formatted by flow? e.g. `.json` or inlined into `.html` files?